nifi-commons/nifi-record-path/src/main/java/org/apache/nifi/record/path/functions/Format.java

```diff
@@ -57,14 +66,22 @@ public Format(final RecordPathSegment recordPath, final RecordPathSegment dateFo
                 });
     }
 
-    private java.text.DateFormat getDateFormat(final RecordPathSegment dateFormatSegment, final RecordPathEvaluationContext context) {
+    private java.text.DateFormat getDateFormat(final RecordPathSegment dateFormatSegment, final RecordPathSegment timeZoneID, final RecordPathEvaluationContext context) {
         final String dateFormatString = RecordPathUtils.getFirstStringValue(dateFormatSegment, context);
         if (dateFormatString == null || dateFormatString.isEmpty()) {
             return null;
         }
 
         try {
-            return DataTypeUtils.getDateFormat(dateFormatString);
+            if (timeZoneID == null) {
+                return DataTypeUtils.getDateFormat(dateFormatString);
+            } else {
+                final String timeZoneStr = RecordPathUtils.getFirstStringValue(timeZoneID, context);
+                if (timeZoneStr == null || timeZoneStr.isEmpty()) {
```

Comment:
Should use `StringUtils.isEmpty(String)`

Reply:
It seems there is no `commons-lang3` artifact in `nifi-record-path` module. 
FYI, the code-style of RecordPath Functions is `charsetString == null || charsetString.isEmpty()` and I just followed it.